cobigen/cobigen-core-api/src/main/java/com/devonfw/cobigen/api/exception/DeprecatedMonolithicTemplatesException.java

```diff
@@ -0,0 +1,35 @@
+package com.devonfw.cobigen.api.exception;
+
+/**
+ * Exception that indicates that an old monolithic template structure has been found. For asking if the template
+ * structure should be upgraded later.
+ */
+public class DeprecatedMonolithicTemplatesException extends CobiGenRuntimeException {
```

Comment:
Where is this Exception catched and handled?

Reply:
it's part of ticket #1531 to handle the exception